scripts/dev-starter.sh

```diff
@@ -7,15 +7,29 @@ set -m
 mesg-daemon start &
 daemon=$!
 
+# function that stop child processes
+function stop {
+  echo "stopping daemon"
+  kill $daemon
+  wait $daemon
+  exit 0
+}
+
+# trap both sigint (ctrl+c) and sigterm (OS ask process to be stopped)
+trap stop SIGINT
+trap stop SIGTERM
+
 # wait 5 sec for the daemon to start rpc server
 sleep 5 &
 wait $!
 
 # start lcd
+echo "starting lcd"
 mesg-cli rest-server --laddr tcp://0.0.0.0:1317 &
 lcd=$!
 
 # start orchestrator
+echo "starting orchestrator"
```

Comment:
@antho1404  Are those log disturbing for the reading of the orchestrator log by CLI?

Reply:
I didn't test but it shouldn't as we start to log only when the API is ready and with a tail of 0. Will need confirmation of that.